### ISSUE TITLE
Add your first credential button in credential screen missing 

### ISSUE THREAD

**Issue by knguyenBC** [bcgov/bc-wallet-mobile#1431]
### Description of problem
"Add your first credential button" in credential screen missing accessibility label

### Expected behavior
Should have a "Add your first credential" accessibility label

### Steps to reproduce
- Turn on voice control
- Go to credential list screen
- Empty credentials in list if necessary

### Screenshots and/or log output
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBc0IwIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--6fbf97d966a7d450acf551bd1e67f568e34dad0c/20230906_163720.png)

### Environment
- [ ] Occurs on Android
- [X] Occurs on iOS

Build #: 1197

Android Device Model: 

iOS Device Model: 

### Severity
- [ ] High
- [ ] Medium
- [x] Low
